### ISSUE TITLE
[enhancement] provide a way to install plugins in headless

### ISSUE THREAD

**Issue by lesquoyb** [gama-platform/gama#381]
**Is your request related to a problem? Please describe.**
In some cases, having a user interface is not an option, for example when running a simulation on a cluster. Unfortunately we don't have a proper way to install plugins directly with gama headless, there was a way using the dropins folder, but it only worked in one specific version and stopped working for unknown reasons in the following, and nobody is really able to fix this so far. 
So for models that do require a plugin to execute, we can not easily run them on clusters or similar environments. 

**Describe the improvement you'd like**
A way to install plugin in gama without using the user interface. It could be with some parameters of the binary, for example:
```
Gama - -install-plugin "https://plugin-address.com/path" "plugin-name" 
```
I think we already have the building blocks for this in the GUI. A problem I see is that when installing in GUI we ask to confirm a few things before actually installing so it might not be as easy to transform that into an interactive command line program, but maybe for a first version we can just go with "accept everything" and add options later? 

**Describe alternatives you've considered**
Making dropins folder work again, but I think it's a lot of work for not much more functionality 

**Additional context**
Gama headless is becoming more popular everyday :)

**Comment by RoiArthurB** [comment]
It's already possible to install plugins without GUI interaction using the `dropins` folder (see issue #26 ).

TL;DR: you can add all the `.jar` that you want to install in this `dropins` folder (existing at the root of your GAMA's installation folder), and they'll be automatically loaded when you start GAMA. Note, however, that this needs to download manually the files and move them where you want + you need to be careful to add all the needed files (eg. you need both the extension(s).jar files as well as the related feature.jar files; otherwise none would be loaded).
